modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -832,6 +850,22 @@ private ObjectField _addObjectField(
 		return objectField;
 	}
 
+	private void _addObjectFolderItem(
+			long objectDefinitionId, long objectFolderId, long userId)
+		throws PortalException {
+
+		if (Validator.isNotNull(
+				_objectFolderItemLocalService.fetchObjectFolderItem(
+					new ObjectFolderItemPK(
+						objectDefinitionId, objectFolderId)))) {
```

Comment:
I think Validator.isNotNull() is only used to compare Strings. To verify if an Object is null, we should do: varName == null

Reply:
Agreed